include/hip/hcc_detail/hip_runtime_api.h

```diff
@@ -35,11 +35,14 @@ THE SOFTWARE.
 #define GENERIC_GRID_LAUNCH 1
 #endif
 
+#include <hsa/hsa.h>
```

Comment:
we are getting weird issues with this line running TF on CI. I'll try get more info on this.

Reply:
I was under the impression that this was verified, which was the reason for asking for a PR...

Reply:
it could be an environment issue. don't worry about it too much (yet)